api/views.py

```diff
@@ -1,3 +1,236 @@
 from django.shortcuts import render
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework_simplejwt.tokens import RefreshToken
+from . import models
+from . import serializers
 
 # Create your views here.
+
+# class Participant
+class RegisterView(APIView):
+    permission_classes = [AllowAny]  # Permet l'accès sans authentification
+
+    def post(self, request, *args, **kwargs):
+        serializer = serializers.ParticipantSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response({"message": "Account created successfully!"}, status=status.HTTP_201_CREATED)
```

Comment:
instead of returning a success message, it's better to return the access token
by returning only a success message, the user will have to authenticate after signing up, which isnt a very good UX

Reply:
ok i'll do this then